code/modules/paperwork/folders.dm

```diff
@@ -98,22 +98,17 @@
 	return data
 
 /obj/item/folder/ui_act(action, params)
+	if(action == "examine")
+		var/obj/item/I = locate(params["ref"]) in src
+		if(istype(I))
+			usr.examinate(I)
+			return TRUE
```

Comment:
This should force them to be either a ghost, or !. because otherwise you can examine while an unconcious human